### ISSUE TITLE
Idea | Decentralized grid-connected solar

### ISSUE THREAD

**Issue by m-anish** [eka-foundation/home#24]
This came up in a conversation today with Mikko. We could do an initial estimate of what would be needed to pilot a decentralized grid-connected solar setup.

**Comment by thepoeticdream** [comment]
Upcoming Enphase IQ8 microinverter bears some study.  Looks ideal for rural microgrids but release to market still unknown.

**Comment by m-anish** [comment]
@thepoeticdream let us know if you think there is something that could be tried out this year in Spiti or Zanskar.

**Comment by thepoeticdream** [comment]
It looks like there are a lot of solar installations in the area and it would be helpful to get a better understanding of the solar ecosystem to see how they're implemented and what's working and not and why.  Guessing lead acid battery failure will the popular one for a solar system not working anymore. It would be helpful to know who the solar players are in the area too, the locals who have the skills and knowledge in doing solar as they are critical to sustainability.  The needs for solar changes from the main population areas with an unstable electrical power grid to the rural off-grid villages.  The solar system recipe changes depending on the problem it's meant to solve.  So what is the problem the proposed solar system meant to solve?  Needs to be defined.

Watching [Ladakh's Remotest Village Now Runs Entirely On Its Own Solar Power Grid](https://special.ndtv.com/lighting-the-himalayas-21/video-detail/ladakhs-remotest-village-now-runs-entirely-on-its-own-solar-power-grid-470333) which an impact tourism group brought solar lighting to Shade village Zanskar of 70 people with 11 homes and a community hall.  The story details village life in darkness and having to use kerosene lamps where the kerosene has to be laboriously carried by long trek.  The trip sponsor is Luminous so the "solar DC micro grid" comprised of 5 grids across the village and 8 DC power LED light bulbs in each home is powered by Luminous 12 volt 10 amp SCC1210NM solar charge controller and Luminous solar tubular batteries (100AH capacity?).  A noname? brand Sunsaver MPPT solar  charger was also shown in the video with a Freescale semiconductor logo on the case :)  All the equipment was brought in by horse and interesting to see in the video of battery acid staining the inside of the cardboard box packaging suggesting the state of charge in the 6 cells of the 12 volt battery is no longer equal (unequal amounts of battery acid) which will impact battery life.  Considering the rough terrain the equipment had to travel it's not surprising.  Also of concern would be any impact damage to the solar PV panels on the journey by horse.  From Kargil, it was 4 days of trekking to Shade and is Zanskar's remotest village according to the story.  Shade has a satellite phone of which they use to call for army helicopter in emergencies.  It was evident that the NDTV crew brought a camera quadcopter with them for the panoramic wide shots of Shade village.  Would be a thought if a delivery quadcopter can carry a couple of lithium battery powerbanks with LED lights to "drone drop" solar lighting solutions to a remote village.  Feel sorry for the horses that had to carry all those heavy lead acid batteries.

**Comment by thepoeticdream** [comment]
[Ladakh will soon be home to world's largest solar plant](https://timesofindia.indiatimes.com/india/ladakh-will-soon-be-home-to-worlds-largest-solar-plant/articleshow/67507422.cms)

5000 megawatts, 45,000 crore, 2023 completion.  It also seems "hill councils" will earn rental income for the land where the solar project will reside.  Will such money spur development in the area?Transmission lines will have to be built to carry the power away.  Many implications IF this comes to fruition.

**Comment by mikkokotila** [comment]
As I recollect, Ladakh has invested heavily in a damn, that damn have been operational for sometime, and Ladakh is apparently currently using a small fraction of the available energy. The issue with Ladakh is more on demand rather than supply; there is an exodus of top talent to major cities and abroad. There is very little in terms of production in Ladakh. In Leh up until few years ago there were no local dairy alternatives for the "urbanizing"/"sub-urbanizing" segment of the population. I've never known of a place the produces so little of what it consumes locally.  

The other issue is with consistency. I think anything that will a) becoming a pawn in national energy/geo politics b) is 24/7 delivered and easy to troubleshoot when there are issues. 

One meaningful implementation for micro-solar is monasteries. They often in remote / back of valley situations and may have significant disruptions to power.

@thepoeticdream what do you think, is there "micro grid" we could bring in to a couple of smaller monasteries first, so they share it with nearby houses or monastery quarters. I would be very interested in developing a PoC like that and I think some earmarked funds would be reasonable straightforward to collect for a wider implementation over a longer period of time.

**Comment by mikkokotila** [comment]
> Upcoming Enphase IQ8 microinverter bears some study. Looks ideal for rural microgrids but release to market still unknown.

This looks brilliant. What is the concrete difference between IQ8 and IQ7? Do you have idea about the pricing for these guys?

**Comment by m-anish** [comment]
Do you think we should email the enphase guys? They might be interested in seeing their technology work in one of the remotest, harshest places on earth.

**Comment by thepoeticdream** [comment]
> This looks brilliant. What is the concrete difference between IQ8 and IQ7? Do you have idea about the pricing for these guys?

IQ7 will not provide power when the electrical grid is down. The IQ8 will continue to provide power when the electrical grid is down or it will provide power via solar/battery in a no-grid setup which becomes a real microgrid. The other development is AC modules where the microinverter is packaged to the solar panel which simplifies installation.  Waaree is in a partnership with Enphase and they offer [this](https://www.waaree.com/ac-module)  On pricing don't know but should inquire with Eco-Soch mentioned below

>  Do you think we should email the enphase guys? They might be interested in seeing their technology work in one of the remotest, harshest places on earth.

Think it better to communicate with [Eco Soch](https://www.ecosoch.com/) as they are one of the largest India enphase installers and staff have worked in Auroville doing rural solar installations in Ladakh including for Ecosphere.  We had some communication with them through Phuntsok.

> One meaningful implementation for micro-solar is monasteries. They often in remote / back of valley situations and may have significant disruptions to power.

>@thepoeticdream what do you think, is there "micro grid" we could bring in to a couple of smaller monasteries first, so they share it with nearby houses or monastery quarters. I would be very interested in developing a PoC like that and I think some earmarked funds would be reasonable straightforward to collect for a wider implementation over a longer period of time.

Totally agree monasteries/nunneries good target for solar.  So there will be on-grid monasteries with unstable power grid with frequent power cuts and the off-grid monasteries with no grid power at all.  Then the question if they have diesel generator/big UPS/existing solar setup that failed so re-use the solar panels etc and what sort of electrical wiring infrastructure that may be total mess in wiring layout and dangerous for electrical fire.  And what are their essential daily power requirements?  Do they have an electric water pump (power hungry) to draw water from well that is essential for water supply? etc.  So there would be some homework to characterize a monastery/nunnery to figure out a correct solar solution.  And importantly identifying monk/nun/local to maintain the system for sustainability.  Anecdotally what I've seen in India are many solar failures and to reduce that, someone with proper skills needs to maintain the system.

But skipping all that work - am guessing the most popular usage case across all of them would be light to read their religious texts or audio/video of teachings off phone/tablet/media player?  In such a scenario feel a bunch of high capacity powerbanks make sense and re-charge them via solar.  There are many forms of USB led lights and evermore expanding market of USB accessories that can be plugged into a powerbank.  Would also figure many monks and nuns familiar with the powerbank as they have already have one and don't think much training involved in order to teach how to charge them via solar.

Would be efficient to distribute at a teaching where they will be making the trek from their remote location.

Powerbank can also be the power source for the EKA wireless nodes :)

**Comment by thepoeticdream** [comment]
Enphase reported their latest financials today.  Also mentioned shipping IQ8 microinverter this quarter to India and Africa to mystery partner that put up the money for it's design.  The IQ8 may be available for general release to market around fourth quarter.

So very well could start seeing the IQ8 microinverter showing up in the India solar landscape soon deployed by the mystery partner.

**Comment by thepoeticdream** [comment]
[BRINGING LIGHT AT 13,000 FEET: LADAKH’S FIRST SOLAR ENGINEERS PROVIDE OFF-GRID ENERGY ACCESS TO ONE OF THE MOST REMOTE AREA OF THE WORLD](https://www.barefootcollege.org/ladakh-global-himalayan-expedition/)

Barefoot College in collaboration with GHE trained women solar engineers Yangchan and Angmo of Ladakh who are solar electrifying villages in Ladakh.  They were trained at Barefoot College's Tilonia facility for 6 months. Rangdum Monastery of around 40 monks was electrified by them.  "Electrified" for Rangdum Monastery seems to be solar powered lighting.  A streetlight pole with light setup in monastery courtyard and presumably lightbulbs throughout monastery.  This was done in 2017?

**Comment by m-anish** [comment]
Related to #25

**Comment by mikkokotila** [comment]
@thepoeticdream any idea what IQ8 price range is?

**Comment by thepoeticdream** [comment]
No.  The release of the off-grid version of the Enphase IQ8 to general market may happen Q4 2019 or later so price is unknown.

**Comment by m-anish** [comment]
@thepoeticdream could you list potential partners we could reach out to?

**Comment by m-anish** [comment]
@m-anish characterize power budgets, and all power needs, especially in winter.

**Comment by holta** [comment]
Right-sizing solar depends so much on local supply chains and widely varying cultural norms of local electricians.

Many Internet-in-a-Box (named "XSCE" at the time) volunteers deployed solar in Haiti, Ghana, Madagascar and other places around the 2012-2015 timeframe, and did excellent solar field work &mdash; but at the same time many took a step back after that &mdash; the thinking was (1) many overseas contributors can have a higher impact nurturing  [equally urgent] Internet-in-a-Box software/usability and (2) letting locals fully run the [hardware] show is often a better way forward.

i.e. Starting solar outfitting businesses etc gives them a business they (locals) ought to be fully in charge of, maximizing local knowledge via local supply chains & local revenue opportunities, even if not always using the very latest solar techniques &mdash; at least they have interchangeable parts and much higher odds for sustainable in-country skillsharing.

Still it would be great to assist those in all countries persisting in these valiant efforts, where common threads/lessons apply to all &mdash; and what fish oil sales/scams to be wary of year-by-year :)

Background:
- http://FAQ.IIAB.IO "How do I provide Solar Power to my school, library, clinic or orphanage?"
- http://wiki.laptop.org/go/IIAB/FAQ#How_do_I_provide_Solar_Power_to_my_school.2C_library.2C_clinic_or_orphanage.3F

Summary / not to be cynical but to be honest: "boutique solar deployments/interventions" can indeed work wonders in isolated communities...and occasionally spark vital innovation...but only occasionally/rarely do they grow sustainably and impactfully beyond hyperlocal "Hero Worship" :/

**Comment by mikkokotila** [comment]
I have also the question related with safety. Energy grid has some unique questions not present in the communications network. Is there a model / framework / set of best practices we can follow?

**Comment by mikkokotila** [comment]
https://www.solaredge.com/solutions/grid-services

**Comment by mikkokotila** [comment]
https://www.sma.de/en/products/smart-module-technology/ts4-r.html

Apparently there is something coming from Huawei as well.

**Comment by thepoeticdream** [comment]
>Summary / not to be cynical but to be honest: "boutique solar deployments/interventions" can indeed work wonders in isolated communities...and occasionally spark vital innovation...but only occasionally/rarely do they grow sustainably and impactfully beyond hyperlocal "Hero Worship" :/

Understanding the regional solar ecosystem supply chain and local human resources is vital for sustainability and needs to be focussed in EKA solar efforts

**Comment by thepoeticdream** [comment]
>I have also the question related with safety. Energy grid has some unique questions not present in the communications network. Is there a model / framework / set of best practices we can follow?

Safety is extremely important as there are inherent dangers involved in different types of solar systems.

I see there is the [Indian National Electrical Code](https://archive.org/details/gov.law.is.nec.2011) which has a solar section in Part 8.

A solar microinverter setup can go up to 220VAC.  A solar string inverter setup can go up to several hundred volts DC.  These are dangerous voltages and improper wiring etc. can expose people to high risk/danger.

Whichever solar system installed will have safety guidelines in the installation guide and bears careful reading and understanding.

Lead acid/Li-Ion batteries are dangerous too and referring to the Manufacturer Material Safety Data Sheet is important.

**Comment by thepoeticdream** [comment]
> https://www.solaredge.com/solutions/grid-services 

This [blog post](https://mcelectrical.com.au/blog/solaredge-inverter-optimiser-review/) has some interesting perspectives on SolarEdge failures in Australia
